packages/gestalt/src/boxTypes.js

```diff
@@ -82,6 +82,7 @@ export type Position = 'static' | 'absolute' | 'relative' | 'fixed';
 export type Right = boolean;
 export type Role = string;
 export type Rounding = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 'circle' | 'pill';
+export type ScrollableOverflow = 'scroll' | 'scrollX' | 'scrollY' | 'auto';
```

Comment:
It doesn't look like this type is used in Box, so this file is an odd location for it. It doesn't look like this is used anywhere outside of ScrollableContainer, so this should just be defined there?

Reply:
DONE